src/libs/Permissions.ts

```diff
@@ -40,6 +40,11 @@ function canUseWorkspaceRules(betas: OnyxEntry<Beta[]>): boolean {
     return !!betas?.includes(CONST.BETAS.WORKSPACE_RULES) || canUseAllBetas(betas);
 }
 
+function canUseCombinedCreateExpense(betas: OnyxEntry<Beta[]>): boolean {
+    // We don't need to show this to all betas.
```

Comment:
Context: we mainly need this beta for developing the feature. We will likely develop a new system for handling A/B testing